missing dependency in velocity_controllers on urdf is breaking build

I believe it was relying on an implicit dependency that has been cleaned up. 

http://jenkins.ros.org/view/IbinS32/job/ros-indigo-velocity-controllers_binarydeb_saucy_i386/25/console

```
[ 33%] Building CXX object CMakeFiles/velocity_controllers.dir/src/joint_velocity_controller.cpp.o
/usr/lib/ccache/c++   -DROSCONSOLE_BACKEND_LOG4CXX -DROS_PACKAGE_NAME=\"velocity_controllers\" -Dvelocity_controllers_EXPORTS -g -O2 -fstack-protector --param=ssp-buffer-size=4 -Wformat -Werror=format-security -D_FORTIFY_SOURCE=2  -O2 -g -DNDEBUG -fPIC -I/tmp/buildd/ros-indigo-velocity-controllers-0.9.0-0saucy-20141102-0505/include -I/opt/ros/indigo/include    -o CMakeFiles/velocity_controllers.dir/src/joint_velocity_controller.cpp.o -c /tmp/buildd/ros-indigo-velocity-controllers-0.9.0-0saucy-20141102-0505/src/joint_velocity_controller.cpp
/usr/bin/cmake -E cmake_progress_report /tmp/buildd/ros-indigo-velocity-controllers-0.9.0-0saucy-20141102-0505/obj-i686-linux-gnu/CMakeFiles 4
[ 66%] Building CXX object CMakeFiles/velocity_controllers.dir/src/joint_position_controller.cpp.o
/usr/lib/ccache/c++   -DROSCONSOLE_BACKEND_LOG4CXX -DROS_PACKAGE_NAME=\"velocity_controllers\" -Dvelocity_controllers_EXPORTS -g -O2 -fstack-protector --param=ssp-buffer-size=4 -Wformat -Werror=format-security -D_FORTIFY_SOURCE=2  -O2 -g -DNDEBUG -fPIC -I/tmp/buildd/ros-indigo-velocity-controllers-0.9.0-0saucy-20141102-0505/include -I/opt/ros/indigo/include    -o CMakeFiles/velocity_controllers.dir/src/joint_position_controller.cpp.o -c /tmp/buildd/ros-indigo-velocity-controllers-0.9.0-0saucy-20141102-0505/src/joint_position_controller.cpp
In file included from /tmp/buildd/ros-indigo-velocity-controllers-0.9.0-0saucy-20141102-0505/src/joint_position_controller.cpp:43:0:
/tmp/buildd/ros-indigo-velocity-controllers-0.9.0-0saucy-20141102-0505/include/velocity_controllers/joint_position_controller.h:70:24: fatal error: urdf/model.h: No such file or directory
 #include <urdf/model.h>
                        ^
compilation terminated.
```

--- a/velocity_controllers/CMakeLists.txt
+++ b/velocity_controllers/CMakeLists.txt
@@ -3,19 +3,25 @@ project(velocity_controllers)
 
 # Load catkin and all dependencies required for this package
 find_package(catkin REQUIRED COMPONENTS
+  angles
+  control_msgs
+  control_toolbox
   controller_interface
   forward_command_controller
-  control_toolbox
   realtime_tools
+  urdf
 )
 
 # Declare catkin package
 catkin_package(
-  CATKIN_DEPENDS 
+  CATKIN_DEPENDS
+    angles
+    control_msgs
+    control_toolbox
     controller_interface
     forward_command_controller
-    control_toolbox
     realtime_tools
+    urdf
   INCLUDE_DIRS include
   LIBRARIES ${PROJECT_NAME}
 )

--- a/velocity_controllers/package.xml
+++ b/velocity_controllers/package.xml
@@ -15,15 +15,21 @@
 
   <buildtool_depend>catkin</buildtool_depend>
 
+  <build_depend>angles</build_depend>
+  <build_depend>control_msgs</build_depend>
+  <build_depend>control_toolbox</build_depend>
   <build_depend>controller_interface</build_depend>
   <build_depend>forward_command_controller</build_depend>
-  <build_depend>control_toolbox</build_depend>
   <build_depend>realtime_tools</build_depend>
+  <build_depend>urdf</build_depend> 
 
+  <run_depend>angles</run_depend>
+  <run_depend>control_msgs</run_depend>
+  <run_depend>control_toolbox</run_depend>
   <run_depend>controller_interface</run_depend>
   <run_depend>forward_command_controller</run_depend>
-  <run_depend>control_toolbox</run_depend>
   <run_depend>realtime_tools</run_depend>
+  <run_depend>urdf</run_depend>
 
   <export>
     <controller_interface plugin="${prefix}/velocity_controllers_plugins.xml"/>

